### PR TITLE
chore(ci): skip preview if no relevant code changes

### PR DESCRIPTION
**Pull request by shulaoda**
## Summary

Related to #7757 

## Checklist

- [ ] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

File: .github/workflows/preview-commit.yml [hunk 1]
```diff
@@ -87,5 +87,4 @@ jobs:
         run: ls -R npm
 
       - name: Release
-        if: needs.check-changed.outputs.changed == 'true'
```
Comment (chenjiahan) [inline .github/workflows/preview-commit.yml:90]: Does this means this action no longer skip the Release job?

Comment (shulaoda) [reply .github/workflows/preview-commit.yml:90]: Noop, just advanced the skipped judgment to the build job

Comment (shulaoda) [reply .github/workflows/preview-commit.yml:90]: If the release is skipped, it should not be built

Comment (chenjiahan) [reply .github/workflows/preview-commit.yml:90]: get👌

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**